Classes/UT3Cicada.uc

```diff
@@ -240,20 +240,60 @@ defaultproperties
     //==========END==========
     VehicleNameString = "UT3 Cicada"
 
-    GroundSpeed=2000
+    GroundSpeed=1600
+    MaxRandForce=2.0
+    RandForceInterval=0.95
+    RollTorqueMax=60 //100
+    RollTorqueStrafeFactor=130 /100
+    RollTorqueTurnFactor=250 //750
+    RollDamping=100.0 //30.0
+    PitchTorqueFactor=50.0 //200.0
+    PitchTorqueMax=20.0 //35.0
+    TurnTorqueFactor=600.0
+    TurnTorqueMax=220.0 //200.0
+    TurnDamping=65.0 //50.0
+    MaxYawRate=1.8 //1.5
+    MaxRiseForce=130.0 //200
+    UpDamping=0.08 //0.05
+    MaxStrafeForce=45.0 //65.0
+    LatDamping=0.08 //0.05
+    MaxThrustForce=80.0 //80.0
+    LongDamping=0.8 //0.3
     CollisionHeight=70.000000
     //DriverWeapons(0)=(WeaponClass=class'UT3CicadaMissileLauncher',WeaponBone=RightRLAttach);
     //DriverWeapons(1)=(WeaponClass=class'UT3CicadaMissileLauncher',WeaponBone=LeftRLAttach);
     TargetLockSound=sound'UT3A_Vehicle_Cicada.Sounds.A_Vehicle_Cicada_TargetLock01'
     IdleSound=Sound'UT3A_Vehicle_Cicada.Sounds.A_Vehicle_Cicada_EngineLoop02'
     StartUpSound=Sound'UT3A_Vehicle_Cicada.Sounds.A_Vehicle_Cicada_Start01'
     ShutDownSound=Sound'UT3A_Vehicle_Cicada.Sounds.A_Vehicle_Cicada_Stop01'//8/10
+    ImpactDamageMult = 0.00003 //0.0003
+    DamagedEffectHealthSmokeFactor=0.65 //0.5
+    DamagedEffectHealthFireFactor=0.38 //0.25
+    DamagedEffectFireDamagePerSec=0.95 //0.75
```

Comment:
As odd as it may be, it is in fact 2.0 in the code: http://greatemerald.xmpcommunity.com/uncodex/UT3-15/Source_utgame/utvehicle.html#7021

Reply:
So it wasn't my imagination some are losing 2 points of health per second after all, it's really hard to tell from actual game play between the get in and get out time it takes damage so quickly in UT3